app/dao/RouteDao.scala

```diff
@@ -0,0 +1,134 @@
+package dao
+
+import java.util
+
+import dao.implicits.DateTimeTransform._
+import dao.implicits.IdTransform._
+import dao.implicits.RouteTransform._
+import dao.mapping.RoutePoint
+import dao.repo.RouteRepo
+import javax.inject.{Inject, Singleton}
+import org.joda.time.DateTime
+import play.api.Logging
+
+import scala.collection.mutable.ListBuffer
+import scala.concurrent.{ExecutionContext, Future}
+import scala.jdk.CollectionConverters._
+
+@Singleton
+class RouteDao @Inject()(routeRepo: RouteRepo)(implicit ec: ExecutionContext)
+  extends BaseDao[mapping.Route, Long, RouteRepo](routeRepo)(ec) with Logging {
+
+  override def ensureExists(): Future[Unit] = {
+    (for {
+      _ <- routeRepo.createTable()
+      _ <- routeRepo.createPointsTable()
+      _ <- routeRepo.createRoutePointsTable()
+    } yield Future.successful(())).flatMap { _ =>
+      routeRepo.getPoint(1).flatMap {
+        case None =>
+          for {
+            p1 <- createPoint(mapping.Point(None, 10.0, 10.0, "first"))
+            p2 <- createPoint(mapping.Point(None, 20.0, 20.0, "second"))
```

Comment:
`p1`, `p2` meeh

Reply:
It's like 3-line scope where it's clear that these are just temp points